flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/FlinkInputSplit.java

```diff
@@ -19,20 +19,17 @@
 
 package org.apache.iceberg.flink.source;
 
-import org.apache.flink.core.io.InputSplit;
 import org.apache.flink.core.io.LocatableInputSplit;
 import org.apache.iceberg.CombinedScanTask;
 import org.apache.iceberg.relocated.com.google.common.base.MoreObjects;
 
-/**
- * TODO Implement {@link LocatableInputSplit}.
- */
-public class FlinkInputSplit implements InputSplit {
+public class FlinkInputSplit extends LocatableInputSplit {
 
   private final int splitNumber;
```

Comment:
we can remove `splitNumber` since it is in the base class now.

Reply:
thx. done.